src/stories/Library/recommender/recommender.scss

```diff
@@ -0,0 +1,83 @@
+.recommender {
+  background-color: $c-global-tertiary-2;
+
+  @include breakpoint-m {
+    padding: $s-6xl;
+  }
+
+  h1 {
+    color: $c-text-primary-white;
+    text-align: center;
+    padding: $s-lg;
+
+    @include breakpoint-m {
+      padding-bottom: $s-6xl;
+    }
+  }
+}
+
+.recommender-grid {
+  display: grid;
+  grid-template-columns: 50% 50%;
+  grid-template-rows: 50% 50%;
+
+  @include breakpoint-m {
+    grid-template-columns: 33.33% 33.33% 33.33%;
+  }
+}
+
+.recommender__grid__material {
+  border: 1px solid $c-text-secondary-gray;
+  padding: 30px;
+  display: flex;
+  width: 100%;
+  position: relative;
+  height: 350px;
+
+  .button-favourite {
+    cursor: pointer;
+    position: absolute;
+    top: 10px;
+    right: 10px;
```

Comment:
Please consider if this positioning can be reworked as this is something we would like to avoid with BEM.

A couple of suggestions for doing so:

1. Supporting adding additional classes to the `button-favourite` component so you can add `recommender__favourite` class with positioning.
2. Adding a wrapper `<div class="recommender__favourite">`
3. Perhaps it is even possible to do something fancy with CSS layouts.

Reply:
I'll fix